front/app/utils/moduleUtils.tsx

```diff
@@ -366,7 +366,7 @@ export interface OutletsPropertyMap {
     customizedButtonConfig?: CustomizedButtonConfig;
     buttonStyle: BannerButtonStyle;
   };
-  'app.components.SignUpIn.AuthProviders.ContainerEnd': {
+  'app.components.SignUpIn.AuthProviders.ContainerStart': {
```

Comment:
Vienna SSO button is the only one that uses the outlet currently, so we can safely rename it.